osu.Game.Rulesets.Taiko/UI/TaikoPlayfield.cs

```diff
@@ -185,7 +185,6 @@ internal void OnNewResult(DrawableHitObject judgedObject, JudgementResult result
                     var drawableTick = (DrawableDrumRollTick)judgedObject;
 
                     addDrumRollHit(drawableTick);
-                    addExplosion(drawableTick, drawableTick.JudgementType);
```

Comment:
I think we want this back, drum rolls feels empty otherwise (no explosion, no kiai, just bouncing objects...).

Reply:
that's how they are on stable, and how i'd expect them to work.

if we added it back, what would you want? an exception for the default skin?

Reply:
Yeah well stage doesn't have the kiai flashes either, are we going to remove those too?

That is to say, yes, I expect the default skin to be an exception. You could have a skinnable "DrumHitExplosion" which the legacy skin would just return an empty drawable for (or null with allowFallback = false).

Reply:
hmm, i disagree and felt the previous flashes were an oversight. they are far too big for every tick (this isn't something we do in any other game mode). can we reimplement in a better way, ie. tick explosions, at a later point?

Reply:
I guess...